lib/engine/assignable.rb

```diff
@@ -10,7 +12,7 @@ def assigned?(key)
       assignments.key?(key)
     end
 
-    def assign!(key, value = true)
```

Comment:
1870 is using the original signature of this method. You'll have to figure out what its doing with it and make sure not to break it.

Reply:
Adding stack_group: to not mess with the value